src/Core/Components/ProfileMenu/FluentProfileMenu.razor.cs

```diff
@@ -0,0 +1,117 @@
+// ------------------------------------------------------------------------
+// MIT License - Copyright (c) Microsoft Corporation. All rights reserved.
+// ------------------------------------------------------------------------
+
+using Microsoft.AspNetCore.Components;
+
+namespace Microsoft.FluentUI.AspNetCore.Components;
+
+public partial class FluentProfileMenu : FluentComponentBase
+{
+    public FluentProfileMenu()
+    {
+        Id = Identifier.NewId();
+    }
+
+    private bool PopoverVisible { get; set; } = false;
+
+    /// <summary>
+    /// Gets or sets the content to be displayed in the header section of the popover.
+    /// Using this property will override the <see cref="CompanyName" /> and <see cref="SignOutLabel"/> properties.
+    /// </summary>
+    [Parameter]
+    public RenderFragment? HeaderTemplate { get; set; }
+
+    /// <summary>
+    /// Gets or sets the content to be displayed in the main section (body) of the popover.
+    /// Using this property will override the <see cref="EMail"/>, <see cref="FullName"/>, <see cref="Image"/> properties.
+    /// </summary>
+    [Parameter]
+    public RenderFragment? ChildContent { get; set; }
+
+    /// <summary>
+    /// Gets or sets the content to be displayed in the footer section of the popover.
+    /// Using this property will override the <see cref="ViewAccountLabel" /> property.
+    /// </summary>
+    [Parameter]
+    public RenderFragment? FooterTemplate { get; set; }
+
+    /// <summary>
+    /// Gets or sets the status to show. See <see cref="PresenceStatus"/> for options.
+    /// </summary>
+    [Parameter]
+    public PresenceStatus? Status { get; set; }
+
+    /// <summary>
+    /// Gets or sets the title to show on hover. If not provided, the status will be used.
+    /// </summary>
+    [Parameter]
+    public string? StatusTitle { get; set; }
+
+    /// <summary>
+    /// Gets or sets the initials to display if no image is provided.
+    /// By default, the first letters of the <see cref="FullName"/> is used.
+    /// </summary>
+    [Parameter]
+    public string? Initials { get; set; }
+
+    /// <summary>
+    /// Gets or sets the name to display.
+    /// </summary>
+    [Parameter]
+    public string? FullName { get; set; }
+
+    /// <summary>
+    /// Gets or sets the company name to display.
+    /// </summary>
+    [Parameter]
+    public string? CompanyName { get; set; }
```

Comment:
Should this perhaps be a bit more general like `Details`?

Reply:
Use an object? Yes, that makes sense. It could be more flexible.

Reply:
I was thinking more like 'Address Line 1' (general) instead of 'Street' (specific) (where it would still be a string)  but making it an object would offer even more flexibility, yes.

Reply:
I renamed to these new attribute names.
- `HeaderLabel`: Gets or sets the header label (e.g Company Name) to display on the top-left.
- `HeaderButton`: Gets or sets the Header Button label (e.g. Sign out) on the top-right.
- `FooterLabel`: Gets or sets the footer label to display on the bottom-left.
- `FooterLink`: Gets or sets the Footer hyperlink label (e.g. View account) on the bottom-right.